src/tools/r2rdump/R2RMethod.cs

```diff
@@ -214,7 +214,7 @@ public class R2RMethod
         /// <summary>
         /// ECMA metadata reader for the method module.
         /// </summary>
-        public MetadataReader MetadataReader { get; }
+        public R2RReader R2RReader { get; }
```

Comment:
Could you update documentation that used to describe the MetadataReader property?

Reply:
Fixed in 2nd commit, thanks for spotting!